File: example/src/main/java/tr/com/emrememis/sample/MainActivity.java
package tr.com.emrememis.sample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

import tr.com.emrememis.getdeviceid.DeviceInfo;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        DeviceInfo deviceInfo = new DeviceInfo(this);
        Log.d("onEmre",deviceInfo.getDeviceIMEI());
        Log.d("onEmre",deviceInfo.getSecureAndroidId());
        Log.d("onEmre",deviceInfo.getSimSerialNumber());
    }
}

File: example/src/main/java/tr/com/emrememis/sample/MainActivity.kt
package tr.com.emrememis.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import tr.com.emrememis.device.info.library.DeviceInfo

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val deviceInfo = DeviceInfo(this)

        deviceInfo.imei?.let { imei ->
            // your code..
        }
        
        deviceInfo.simSerialNumber?.let { simSerialNumber ->
            // your code..
        }

        deviceInfo.secureId?.let { secureId ->
            // your code..
        }
    }
}
